SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -362,14 +357,6 @@ protected MemoryCachedLocalStorage createCachedLocalStorage() {
         return new MemoryCachedLocalStorage(new MemoryLocalStorage(), null);
     }
 
-    protected ExecutorService createStorageExecutor() {
```

Comment:
Something must of used it at one stage, but nothing was using it anymore. So works fine.